lib/Core/Executor.cpp

```diff
@@ -4201,11 +4219,16 @@ void Executor::executeMemoryOperation(ExecutionState &state,
           terminateStateOnError(state, "memory error: object read only",
                                 ReadOnly);
         } else {
+          klee_message("write to segment: %lu address: %lu", cast<klee::ConstantExpr>(segment)->getZExtValue(), cast<klee::ConstantExpr>(offset)->getZExtValue());
+          klee_message("wrote value %s", getKValueInfo(state, value).c_str());
           ObjectState *wos = state.addressSpace.getWriteable(mo, os);
           wos->write(offset, value);
+
```

Comment:
Why adding the new line here? If you want to do that, it should be in a refactoring-related commit.